Makefile

```diff
@@ -164,7 +164,7 @@ ifndef ignore-not-found
   ignore-not-found = false
 endif
 
-clean-manifests = (cd config/components/manager && $(KUSTOMIZE) edit set image controller=gcr.io/k8s-staging-kueue/kueue:main)
+clean-manifests = (cd config/components/manager && $(KUSTOMIZE) edit set image controller=gcr.io/k8s-staging-kueue/kueue:release-0.2)
```

Comment:
Why moving from main here?

Reply:
Oops wrong target branch. This is for the release-0.2 branch.

Reply:
aaaahhhh ;P